.github/workflows/actions.yml

```diff
@@ -43,8 +43,7 @@ jobs:
         with:
           perl-version: ${{ matrix.perl }}
       - run: perl -V
-      - run: cpanm --quiet --notest --installdeps .
-      - run: cpanm --quiet --notest Test::Pod Test::Spelling Test::Perl::Critic
+      - run: cpanm --quiet --notest --installdeps --with-recommends .
       - run: perl Makefile.PL
```

Comment:
This should run dzil test or something similar as there is no Makefile.PL any more.

Reply:
Thanks for taking a look @abraxxa! The integration test is done against the built tarball. By this stage, we have a Makefile.PL, and anything else dzil generates.

Reply:
I see, thanks for the clarification!